src/cpu_profiler.cc

```diff
@@ -103,9 +105,11 @@ namespace nodex {
 #if (NODE_MODULE_VERSION > 0x0039)
   const_cast<CpuProfile*>(profile)->Delete();
   --data->m_startedProfilesCount;
-  if (!data->m_startedProfilesCount) {
+
+  if (!data->m_startedProfilesCount && data->m_profilesSinceLastCleanup > 1000) {
```

Comment:
We should probably be able to tune this `1000`, probably easiest from environment variable if that's okay with you?

Reply:
So when it is set to 1000, what is the performance comparison before and after this patch?
I think if they are almost the same, we can set an empirical value instead of the user’s perception of this value.